test/integration/webauthn_credentials_test.rb

```diff
@@ -0,0 +1,51 @@
+require "test_helper"
+
+class WebauthnCredentialsTest < SystemTest
+  setup do
+    @user = create(:user)
+  end
+
+  def sign_in
+    visit sign_in_path
+    fill_in "Email or Username", with: @user.reload.email
+    fill_in "Password", with: @user.password
+    click_button "Sign in"
+  end
+
+  should "have security device form" do
```

Comment:
@jenshenny Since we're planning to split the webauthn and mfa settings out of the settings page does it make sense to keep this where it is for now and then move them when these things move to their final home?

Reply:
If we're committed to move the form outside the general settings page, I'm ok with keeping this where it is now 👍